sdcm/cluster.py

```diff
@@ -3437,7 +3437,13 @@ def node_setup(self, node, verbose=False, timeout=3600):
             node.remoter.run('sudo cat /etc/scylla.d/io.conf')
 
             if self.params.get('use_mgmt', None):
-                region = self.params.get('region_name').split()
+                cluster_backend = self.params.get("cluster_backend")
+                if cluster_backend == "aws":
+                    region = self.params.get('region_name').split()
+                elif cluster_backend == "gce":
```

Comment:
I wouldn't support gce here, this would configure the agents wrongly, we aren't support manager backup with gce region, we only support backup to s3, and we didn't integrated this kind of setup in SCT yet

Reply:
the call to `update_config_file` should support region=None. and then backups won't be working (we are no running nemesis in GCE anyhow, for now)

Reply:
hmmm... so what you are telling that we can't use manager backup, when running in non-AWS machines? Can we have some default AWS region for manager backups?

Reply:
@fgelcer, might know better, about what was/is the plan for testing such a thing

But I think there no point support manager backup on GCE, at least until someone would be able to do the integration work needed.
right now we are using attach a role to the ec2 instances to grant them permission to S3, with GCE we'll need to create and pass those in a different way, and selecting the region should have it's own logic, which I have no idea how it would be.
also down the road, manager should support backups to GCE, and that would a whole different story.

Reply:
while we are it, we might want to limit the backup nemesis only to AWS backend, until proven otherwise.
I guess once you'll have images for GCE, we'll want to run few longevity tests, we'll need to skip backup for the time being.

Reply:
got it. Removed GCE support